packages/web-service/src/pages/habitat/confirm-delete/confirm-delete.js

```diff
@@ -0,0 +1,33 @@
+import Joi from 'joi'
```

Comment:
When I enter without selecting and get a validation error, my sett name disappears

Reply:
I think there is a bug here, after deleting I got the 'Sorry, there is a problem with the service' page